ui/src/overlays/reducers/overlays.ts

```diff
@@ -13,6 +13,7 @@ export type OverlayID =
   | 'telegraf-output'
   | 'switch-organizations'
   | 'create-bucket'
+  | 'asset-limit'
 
 export interface OverlayParams {
   [key: string]: string
```

Comment:
I really don't like that these params are actually required for the overlay to load properly, but they are not properly typed. I know you didn't write this, and I don't think you need to fix this in this PR, but it makes me worry :(

Reply:
Maybe we should open another tech debt issue for this.